.pipeline/blubber.yaml

```diff
@@ -0,0 +1,29 @@
+version: v4
+base: docker-registry.wikimedia.org/golang:1.14-1
+lives:
+  in: /srv/service/
+runs:
+  environment:
+    APP_BASE_PATH: /srv/service
+
+variants:
+  build:
+    apt: { packages: [build-essential] }
+    copies: [local]
+    entrypoint: [make, build]
+    runs: { insecurely: false }
```

Comment:
I _think_ we can omit this line, i believe it defaults to false

Reply:
Ack. I was not sure about that by reading the doc. Should have check the code. Thanks!